packages/material-ui/src/Modal/ModalManager.ts

```diff
@@ -13,36 +17,40 @@ function isOverflowing(container) {
   return container.scrollHeight > container.clientHeight;
 }
 
-export function ariaHidden(node, show) {
+export function ariaHidden(node: Element, show: boolean): void {
```

Comment:
```suggestion
export function ariaHidden(element: Element, show: boolean): void {
```

and probably want to visit the other sites as well. It's not as important with types but that the distinction is important is clearer with this PR.

Reply:
done